cmd/oras/internal/option/target.go

```diff
@@ -269,3 +311,11 @@ func (opts *BinaryTarget) Parse() error {
 	opts.To.resolveFlag = append(opts.resolveFlag, opts.To.resolveFlag...)
 	return Parse(opts)
 }
+
+// Handle handles error during cmd execution.
+func (opts *BinaryTarget) Handle(err error, cmd *cobra.Command) error {
+	if err := opts.From.Handle(err, cmd); err != nil {
+		return err
+	}
+	return opts.To.Handle(err, cmd)
```

Comment:
Can we get line 320 covered?

Reply:
Marked. Will add test to make it covered once the refactoring is done.